src/cascadia/TerminalSettingsEditor/KeyChordConverter.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "NullCheckToVisibilityConverter.g.h"
+#include "../inc/cppwinrt_utils.h"
+
+DECLARE_CONVERTER(winrt::Microsoft::Terminal::Settings::Editor, NullCheckToVisibilityConverter);

```

Comment:
is there any way for us to simply stop introducing new converters and just like... use functions?

Reply:
wait - no xaml files exist in this pull request that use this converter.

Reply:
Whoops! Missed that file from an earlier iteration, it seems.